rpm/cray/csm/sle-15sp4/index.yaml

```diff
@@ -42,4 +42,4 @@ https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp4/:
     - metal-ipxe-2.2.14-1.noarch
     - pit-init-1.2.38-1.noarch
```

Comment:
Based on other PRs it looks like this should be updated to pit-init version 1.2.43-1,  no?
https://github.com/Cray-HPE/csm-rpms/pull/714/files